source/release-notes.txt

```diff
@@ -10,6 +10,39 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.37.0
+----------------
+
+*Released May 25, 2023*
+
+New Features:
+
+- Dark mode has been updated with a modern theme. If the modern theme 
+  is enabled in ``Settings>Feature Preview``, Compass defaults to 
+  the modern dark mode theme
+- When Compass detects a potential error importing a CSV file. The  
+  warning banner now shows the value, type, and issue associated with the error (:issue:`COMPASS-6543`)
```

Comment:
I know @lerouxb wrote a larger write-up on all the changes related to import/export in #compass. This line is accurate, but including it in this way probably gives the impression that this is a standalone change, and not part of a larger revamp?

Reply:
The import/export improvements epic might make sense to link as well in that case: https://jira.mongodb.org/browse/COMPASS-5576

Reply:
Will do